packages/eslint-config-base/rules/exports.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
This should live under the `./rules/imports` file.

Reply:
I was thinking that, but then I realized that all of the logic is specifically around exports, not imports, which made it seem beneficial to group them in separate files. What do you think?

Reply:
I'd like to keep them in the same file since it's dealing with the same lib.

Reply:
Oh, I see! You mean eslint-plugin-import. That makes total sense; I will move the rules into `import.js`